test/mocha.opts

```diff
@@ -0,0 +1,2 @@
+--require ./test/support.js
+--require ./test/helpers/loopback-testing-helper.js
```

Comment:
Why is this second line required? AFAICT, all modules using `loopback-testing-helper` are directly calling `require()` to get it.